Modules/Filtering/ImageGrid/include/itkPasteImageFilter.h

```diff
@@ -159,7 +201,10 @@ class ITK_TEMPLATE_EXPORT PasteImageFilter : public InPlaceImageFilter<TInputIma
 
   SourceImageRegionType m_SourceRegion;
 
-  InputImageIndexType m_DestinationIndex;
+  InputImageIndexType    m_DestinationIndex;
+  InputSkipAxesArrayType m_DestinationSkipAxes;
+
+  static_assert(InputImageDimension >= SourceImageDimension, "The source image is greater than the input image.");
```

Comment:
Should `The source image is greater...` possibly be `The source image dimension is greater...`?